src/utils/daos_hdlr.h

```diff
@@ -175,6 +176,15 @@ int pool_autotest_hdlr(struct cmd_args_s *ap);
  * int pool_stat_hdlr(struct cmd_args_s *ap);
  */
 
+/* general datamover operations */
+void dm_cont_free_usr_attrs(int n, char*** _names, void*** _buffers,
```

Comment:
(style)  "foo*** bar" should be "foo ***bar"
(style)  "foo*** bar" should be "foo ***bar"